moj-node/server/app.js

```diff
@@ -91,6 +93,15 @@ module.exports = function createApp({
   //  Static Resources Configuration
   const cacheControl = { maxAge: config.staticResourceCacheDuration * 1000 };
 
+  app.use(session({
```

Comment:
In order to reduce calls to the api I'm caching the menu calls for 5 minutes in memory